src/e2e/resources/jointParty/hearingReminderCallback.json

```diff
@@ -237,6 +239,66 @@
           }
         ]
       },
+      "hearings": {
+        "classification": "PUBLIC",
+        "value": [
+          {
+            "id": "609523de-ba0a-45e7-8f0b-af3f82ab347c",
+            "value": {
+              "adjourned": "PUBLIC",
+              "hearingDate": "PUBLIC",
+              "time": "PUBLIC",
+              "venue": {
+                "classification": "PUBLIC",
+                "value": {
+                  "address": {
+                    "classification": "PUBLIC",
+                    "value": {
+                      "county": "PUBLIC",
+                      "line1": "PUBLIC",
+                      "line2": "PUBLIC",
+                      "postcode": "PUBLIC",
+                      "town": "PUBLIC"
+                    }
+                  },
+                  "googleMapLink": "PUBLIC",
+                  "name": "PUBLIC"
+                }
+              }
+            }
+          }
+        ]
+      },
+      "hearings": {
```

Comment:
Flagging for future reference that this is duplicated.

Reply:
Thanks. I imagine this file will change drastically in the testing ticket work.